apmpackage/apm/_dev/build/docs/README.md

```diff
@@ -1,74 +1,33 @@
 # APM Integration
 
-The APM integration installs Elasticsearch templates and ingest node pipelines for APM data.
+Elastic APM is an application performance monitoring system built on the Elastic Stack. It allows you to monitor software services and applications in real-time, by collecting detailed performance information on response time for incoming requests, database queries, calls to caches, external HTTP requests, and more.
 
+The APM integration installs templates and pipelines that validate and process APM data.
 To learn more about the APM Integration architecture, see [APM Components](https://ela.st/apm-components).
 
 ### Quick start
 
 Ready to jump in? Read the [APM quick start](https://ela.st/quick-start-apm).
 
-### How to use this integration
+## Data Streams
 
-Add the APM integration to an Elastic Agent policy to create an `apm` input.
-Any Elastic Agents set up with this policy will run an APM Server binary locally.
-Don't forget to configure the APM Server `host`, especially if it needs to be accessed from outside, like when running in Docker.
-Then, configure your APM agents to communicate with APM Server.
-
-If you have Real User Monitoring (RUM) enabled, you must run Elastic Agent centrally.
-Otherwise, you can run it on edge machines by downloading and installing Elastic Agent
-on the same machines that your instrumented services run.
-
-#### Data Streams
-
-When using the APM integration, apm events are indexed into data streams. Data stream names contain the event type,
+When using the APM integration, APM events are indexed into data streams. Data stream names contain the event type,
```

Comment:
> Data stream names contain the event type,

not on a fine granular level (traces, metrics, logs vs. transactions, spans, errors, ..)
How about removing this bit?

Reply:
Good point!